### PR TITLE
PayPal docs

### PR DESCRIPTION
**Pull request by ivov**
Here are the PayPal docs. The new example workflow is [here](https://n8n.io/workflows/438).

The PayPal app creation process is slightly different now, so I also updated the credentials.

Let me know if you have any comments.

### CODE REVIEW

File: docs/nodes/nodes-library/nodes/PayPal/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+You can find authentication information for this node [here](../../../credentials/PayPal/README.md).
+:::
+
+## Basic Operations
+
+- Payout
+	- Create a batch payout
+	- Get payout details
+- Payout item
+	- Canel an unpaid payout item
```
Comment (Shraddha2104) [inline docs/nodes/nodes-library/nodes/PayPal/README.md]: typo- cancel?

File: docs/nodes/nodes-library/nodes/PayPal/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+![A workflow with the PayPal node](./workflow.png)
+
+### 1. Start node
+
+The Start node exists by default when you create a new workflow.
+
+### 2. PayPal node
+
+1. Click on the plus button, search for "PayPal", select it and double-click on the new node.
```
Comment (Shraddha2104) [inline docs/nodes/nodes-library/nodes/PayPal/README.md:38]: you can probably skip this and the next step

File: docs/nodes/nodes-library/nodes/PayPal/README.md (hunk 1 above)
Comment (ivov) [reply docs/nodes/nodes-library/nodes/PayPal/README.md]: Good catch. I should install a spellchecker for markdown.

File: docs/nodes/nodes-library/nodes/PayPal/README.md (hunk 2 above)
Comment (ivov) [reply docs/nodes/nodes-library/nodes/PayPal/README.md:38]: I added it like this in the [Slack node docs](https://github.com/n8n-io/n8n-docs/tree/master/docs/nodes/nodes-library/nodes/Slack) and it was merged.

**Comment by ivov** [comment]
Submitted to maintainers: https://github.com/n8n-io/n8n-docs/pull/12

**Pull request closed without merging**